src/client/eppo-client.ts

```diff
@@ -987,6 +995,34 @@ export default class EppoClient {
     validateNotBlank(subjectKey, 'Invalid argument: subjectKey cannot be blank');
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
+    // Check for override early
+    const overrideVariation = this.overridesStore?.get(flagKey);
+    if (overrideVariation) {
+      const configFormat = this.overridesStore?.getFormat() ?? '';
```

Comment:
this position and the return value of a synthetic match makes sense to me - please add a test to the existing evaluation code to verify and codify the behavior. this file is the suggested location - https://github.com/Eppo-exp/js-sdk-common/blob/main/src/client/eppo-client.spec.ts

Reply:
Added test coverage here https://github.com/Eppo-exp/js-sdk-common/pull/184/commits/cf66589908c577a57426e3f11deb7b68475ddcf7